elibrary-hibernate/src/main/java/com/javafee/elibrary/hibernate/dao/common/Common.java

```diff
@@ -21,6 +16,10 @@
 import com.javafee.elibrary.hibernate.dto.library.Client;
 import com.javafee.elibrary.hibernate.dto.library.LibraryData;
 
+import javax.persistence.TypedQuery;
```

Comment:
Please put all the imports in the places where they were. 
It's related to the all changed files.